java/src/com.zeroc.ice/src/main/java/com/zeroc/Ice/InputStream.java

```diff
@@ -281,93 +111,7 @@ public void clear() {
         _startSeq = -1;
     }
 
-    /**
-     * Sets the value factory manager to use when unmarshaling value instances. If the stream was
-     * initialized with a communicator, the communicator's value factory manager will be used by
-     * default.
```

Comment:
All of the methods below this comment were removed because we don't expect any users to ever call them I guess?

Reply:
They were removed for several reasons:
 - I already removed them in C++ and Java
 - they were not called anywhere (= not tested)
 - they were nominally to allow you to set ValueFactoryManager (etc.) after constructing an InputStream without a communicator (not really to override the one set in the constructors)
 - it's much simpler to make all these fields (_valueFactoryManager, _classGraphMaxSize...) read-only fields with values derived from instance and set only at construction time.